eve/main.yml

```diff
@@ -211,7 +211,7 @@ models:
           controlPlane: 10.100.0.0/16
           workloadPlane: 10.100.0.0/16
         ca:
-          minion: $(hostname)
+          minion: bootstrap
```

Comment:
I think it could make sense to leave one of the tested scenarios to the default, which is hostname for Salt. That's what some, if not most users, will do, probably makes sense to test it regularly.

Reply:
Yes make sense